handlers/src/main/java/com/nextdoor/bender/handler/s3/S3EventIterator.java

```diff
@@ -187,6 +190,17 @@ private void updateCursor() {
       logger.trace("s3 get request id: " + client.getCachedResponseMetadata(req).getRequestId()
           + " host: " + client.getCachedResponseMetadata(req).getHostId() + " cloudfrontid: "
           + client.getCachedResponseMetadata(req).getCloudFrontId());
+      long notificationDelay = obj.getObjectMetadata().getLastModified().getTime()
```

Comment:
This looks fine to me .. I might want to see it customizable in the future - but otherwise this seems fine. @stlava you shoudl give the final approval..